imap_processing/ultra/tests/unit/test_decom_apid_880.py

```diff
@@ -0,0 +1,80 @@
+import pandas as pd
+import pytest
+
+from imap_processing import decom
+
+
+@pytest.fixture()
+def decom_test_data(ccsds_path, xtce_aux_path):
+    """Read test data from file"""
+    data_packet_list = decom.decom_packets(ccsds_path, xtce_aux_path)
+    return data_packet_list
+
+
+def test_ultra_apid_880_length(decom_test_data):
+    """Test if total packets in data file is correct"""
+    total_packets = 23
+    assert len(decom_test_data) == total_packets
+
+
+def test_enumerated_apid_880(decom_test_data):
+    """Test if enumerated values derived correctly"""
+
+    for packet in decom_test_data:
+
+        if packet.data["SPINPERIODVALID"].raw_value == 0:
+            assert packet.data["SPINPERIODVALID"].derived_value == 'INVALID'
+        elif packet.data["SPINPERIODVALID"].raw_value == 1:
+            assert packet.data["SPINPERIODVALID"].derived_value == 'VALID'
+        if packet.data["SPINPHASEVALID"].raw_value == 0:
+            assert packet.data["SPINPHASEVALID"].derived_value == 'INVALID'
+        elif packet.data["SPINPHASEVALID"].raw_value == 1:
+            assert packet.data["SPINPHASEVALID"].derived_value == 'VALID'
+        if packet.data["SPINPERIODSOURCE"].raw_value == 0:
+            assert packet.data["SPINPERIODSOURCE"].derived_value == 'SAFING'
+        elif packet.data["SPINPERIODSOURCE"].raw_value == 1:
+            assert packet.data["SPINPERIODSOURCE"].derived_value == 'NOMINAL'
+        if packet.data["CATBEDHEATERFLAG"].raw_value == 0:
+            assert packet.data["CATBEDHEATERFLAG"].derived_value == 'UNFLAGGED'
+
+
+def test_mode_apid_880(decom_test_data):
```

Comment:
what packet is `apid_880`? Can we use the packet name instead of the appid? Eg. `test_mode_in_image_rate_packet`